doc/faq_troubleshooting.rst

```diff
@@ -0,0 +1,114 @@
+.. SPDX-FileCopyrightText: Contributors to Open-TYNDP <https://github.com/open-energy-transition/open-tyndp>
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _faq_troubleshooting:
+
+###############################################
+FAQ and Troubleshooting
+###############################################
+
+This page contains frequently asked questions about the Open-TYNDP project:
+
+General Questions
+==================
+
+.. admonition:: What is Open-TYNDP?
+
+   Open-TYNDP is an open-source research and innovation project, which is a collaboration between `Open Energy Transition (OET) <https://openenergytransition.org/>`__ and ENTSO-E. The project aims to explore the option of a complementary open-source tool in the Ten-Year Network Development Plan (TYNDP) by building a workflow based on PyPSA-Eur. It provides one streamlined tool for the Scenario Building and the Cost-Benefit Analysis (CBA) of the TYNDP.
```

Comment:
Since we use the abbreviation `sb` also in the repository, it is nice to add it here as well
```suggestion
   Open-TYNDP is an open-source research and innovation project, which is a collaboration between `Open Energy Transition (OET) <https://openenergytransition.org/>`__ and ENTSO-E. The project aims to explore the option of a complementary open-source tool in the Ten-Year Network Development Plan (TYNDP) by building a workflow based on PyPSA-Eur. It provides one streamlined tool for the Scenario Building (SB) and the Cost-Benefit Analysis (CBA) of the TYNDP.
```